dist/base.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
What's the use-case for exposing this file?

Reply:
This is what apps consuming Polaris React will use and pass to their app providers should they want the new design language as Polaris will ship with a generic default theme.